Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -989,27 +958,9 @@ extension SourceKitLSPServer {
   }
 
   func initialize(_ req: InitializeRequest) async throws -> InitializeResult {
-    if case .dictionary(let options) = req.initializationOptions {
-      if case .bool(let listenToUnitEvents) = options["listenToUnitEvents"] {
-        self.options.indexOptions.listenToUnitEvents = listenToUnitEvents
-      }
-      if case .dictionary(let completionOptions) = options["completion"] {
-        switch completionOptions["maxResults"] {
-        case .none:
-          break
-        case .some(.null):
-          self.options.completionOptions.maxResults = nil
-        case .some(.int(let maxResults)):
-          self.options.completionOptions.maxResults = maxResults
-        case .some(let invalid):
-          logger.error("Expected null or int for 'maxResults'; got \(String(reflecting: invalid))")
-        }
-      }
-    }
```

Comment:
I guess this isn't supported anymore?

Reply:
Yes, I’m not aware of any uses for it and I don’t see it as useful. We can always add it again if we should need it, at which point we probably wouldn’t make it in initialization option anymore either.